.gitignore

```diff
@@ -20,6 +20,8 @@ docs/reference/src/language/builtins/structs.md
 *.node
 *.d.ts
 
+api/node/dist/*
```

Comment:
I think this belongs into `api/node/.gitignore`. Or is there a reason for having it here instead of the more specific ignore file?